lib/isaac_state_manager.go

```diff
@@ -0,0 +1,195 @@
+// IsaacStateManager manages the IsaacState.
+// The most important function `Start()` is called in StartStateManager() function in node_runner.go by goroutine.
+
+package sebak
+
+import (
+	"time"
+
+	"boscoin.io/sebak/lib/common"
+	"boscoin.io/sebak/lib/round"
+)
+
+type IsaacStateManager struct {
+	nr             *NodeRunner
+	state          IsaacState
+	conf           *IsaacConfiguration
+	stateTransitCh chan IsaacState
+	nextHeightCh   chan bool
+	stopCh         chan bool
```

Comment:
Why do you want to use system Hungarian notation here ? :'(

Reply:
I didn't know that is Hungarian. I use it because sometimes it is duplicated with method name.
I just can remove `Ch`. No problem.